mobile/app/(onboard)/login.tsx

```diff
@@ -1,48 +1,21 @@
 import { AuthError } from "@supabase/supabase-js";
-import { MoveRight } from "@tamagui/lucide-icons";
-import { Link, router } from "expo-router";
+import { router } from "expo-router";
 import React, { useEffect, useState } from "react";
 import { Alert, SafeAreaView } from "react-native";
 import { Button, YStack, Text, Input } from "tamagui";
 import LoadingScreen from "../../components/atoms/LoadingScreen";
 import DismissKeyboardContainer from "../../components/containers/DismissKeyboardContainer";
 import { FloatingActionBar } from "../../components/containers/TabBar";
 import { supabase } from "../../utils/supabase";
-import AsyncStorage from "@react-native-async-storage/async-storage";
+import PasswordInput from "../../components/atoms/PasswordInput.tsx";
+import EmailInput from "../../components/atoms/Emailnput.tsx";
 
 export default function LoginSignup() {
   const [isLogin, setIsLogin] = useState(false);
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [loading, setLoading] = useState(false);
 
-  // TODO: turn this into a hook
-  const persistsSuccessfullLogin = async (value) => {
```

Comment:
@mashnoon33 just fyi I took this out because I _think_ it's redundant - the supabase client stores the session in local storage (in `supabase.ts`). LMK if I'm missing something tho